### PR TITLE
Adds a new Test ReadFromCSVFileTest to the garage.com project.

### PR DESCRIPTION
**Pull request by renekounkou**

### CODE REVIEW

File: garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+public class ReadFromCSVFileTest {
+  @Test
+  public void readfromCSVFile() throws IOException{
+	  String csvFilePath = "./src/test/resources/basses.csv";
+	  CSVReader reader = new CSVReader(new FileReader(csvFilePath));
+	  List<String[]> data = reader.readAll();
+	  for(String[] row: data) {
+		  System.out.println(Arrays.toString(row));
+		  String expectedResult = csvFilePath;
+		  assertEquals(csvFilePath, expectedResult);
```
Comment (dentaylor) [inline garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java]: Assertion should not be in this loop

File: garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	  }
```
Comment (dentaylor) [inline garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java:23]: Fix indentation

File: garage.com/src/test/java/com/garage/ReadFromCSVFileTest1.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		  
+		  System.out.println(map);
+		  String expectedResult = csvFilePath;
+		  assertEquals(csvFilePath, expectedResult);
+	  }
+	  
+  }
+}
+
+
```
Comment (dentaylor) [inline garage.com/src/test/java/com/garage/ReadFromCSVFileTest1.java:41]: Remove extra lines (43 & 44)

**Review by dentaylor — CHANGES_REQUESTED** [review]

File: garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java (hunk 1 above)
Comment (renekounkou) [reply garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java]: Moved the assertion outside of the loop.

File: garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java (hunk 2 above)
Comment (renekounkou) [reply garage.com/src/test/java/com/garage/ReadFromCSVFileTest.java:23]: Resolved.

File: garage.com/src/test/java/com/garage/ReadFromCSVFileTest1.java (hunk 3 above)
Comment (renekounkou) [reply garage.com/src/test/java/com/garage/ReadFromCSVFileTest1.java:41]: Resolved.

**Review by dentaylor — APPROVED** [review]

**Pull request merged**